docs/tutorials/annotation/annotate_points.md

```diff
@@ -97,14 +101,17 @@ def point_annotator(
     """
     stack = imread(im_path)
 
+    fake_data = [[0.0, 0.0, 0.0]] * len(labels)
+
     viewer = napari.view_image(stack)
     points_layer = viewer.add_points(
+        data=fake_data,
         properties={'label': labels},
         edge_color='label',
         edge_color_cycle=COLOR_CYCLE,
         symbol='o',
         face_color='transparent',
-        edge_width=8,
+        edge_width=0.5,
```

Comment:
The function signature of `add_points` has changed and now needs a relative instead of an absolute size for `edge_width`.

Reply:
Well, optionally, you can pass `edge_width_is_relative=False` to use the old behaviour. tbh I suspect we might revisit this again in the future... CC @brisvag @andy-sweet. For now I would add a comment:

```suggestion
        edge_width=0.5,  # fraction of point size
```

Possibly it should be a bit smaller then, like 0.1?